CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v0.5.0 (2016-03-14)
```

Comment:
Updated below, removed the date for now and can be added again later when 0.5.0 is released.